pallene/checker.lua

```diff
@@ -370,7 +415,7 @@ function Checker:expand_function_returns(rhs)
     end
 end
 
-function Checker:check_stat(stat)
+function Checker:check_stat(stat, toplevel)
```

Comment:
1. I suggest using the name "isXXX" for boolean variables
2. Consider using `check_stat(stat, false)` instead of just `check_stat(stat)`

Reply:
Ok, will do.